src/index.ts

```diff
@@ -15,6 +17,8 @@ export {
   DispatchPlayerContext,
   PersistentPlayer,
   PlayerContext,
+  IcecastPlayer,
```

Comment:
so `IcecastPlayer` is a distinctly different Audio player? 🎧 

Is there a way to unify both components into one, singular, audio player? Or, does that not make sense?

Reply:
yeah it's a stream of live audio, and you can receive server-sent events to receive metadata on the current track. I don't think it makes sense to unify because you have less controls (ex: there is no scrubbing or skipping to next track).  That's why I have all its components under one dir